python/test/unit/language/test_core_amd.py

```diff
@@ -2746,6 +2746,10 @@ def test_dot_mfma_vector_load(vec_size, swizzle, transposeA, transposeB):
     if transposeA and not transposeB:
         pytest.skip()
 
+    # We skip for the case when swizzle is disabled for k-major opA or opB
+    if not swizzle and (not transposeA or transposeB):
+        pytest.skip()
+
```

Comment:
Do you want to speedup testing or just do not test these cases because they should not appear with user code?

If the reason is not about testing speed I suggest to add check in `fastPathAvailable` and re-enable these tests.
Triton users could never run into unswizzled cases, but developers can.

Reply:
This makes sense. I enabled the test.